assets/general/adamant/info.json

```diff
@@ -19,6 +19,9 @@
   "defaultOrdinalLevel": 0,
   "consensus": "dPoS",
   "blockTimeFixed": 5000,
+  "balanceCheckInterval": 5000,
+  "balanceCheckIntervalNewAccount": 2000,
+  "balanceValidInterval": 300000,
```

Comment:
These props must be added to the Swagger shema https://github.com/Adamant-im/adamant-wallets/blob/dev/specification/openapi.json

You can validate the schema here: https://editor.swagger.io/

Reply:
Also update README.md. Describe that these props do.